controller/validate/index.ts

```diff
@@ -28,6 +30,32 @@ const controller = {
     }
   },
 
+  validateFIS: async (req: Request, res: Response): Promise<void> => {
+    try {
+      const { domain, version, payload, flow } = req.body
+      let response
+
+      if (!(flow in fisFlows)) {
+        res.status(400).send({ success: false, response: 'Invalid Flow!!' })
+      } else {
+        switch (version) {
+          case '1.2.0':
```

Comment:
This switch case is for retail version 1.2.0. pls utilize the validate function itself and accommodate all the domains there